src/builder/edit_guild.rs

```diff
@@ -264,4 +264,8 @@ impl EditGuild {
         let num = Value::Number(Number::from(verification_level.num()));
         self.0.insert("verification_level", num);
     }
+
+    pub fn system_channel_flags(&mut self, system_channel_flags: SystemChannelFlags) {
+        self.0.insert("system_channel_flags", system_channel_flags.bits().into());
+    }
```

Comment:
This should return `&mut Self` to allow chaining the builder with other methods. Documentation should also be added.
```suggestion
    pub fn system_channel_flags(&mut self, system_channel_flags: SystemChannelFlags) -> &mut Self {
        self.0.insert("system_channel_flags", system_channel_flags.bits().into());
        self
    }
```

Reply:
Fixed on 1e3f689